lib/utils/mbaas-token-check.js

```diff
@@ -0,0 +1,20 @@
+var common = require("../common");
+var fhreq = require("./request");
+
+module.exports = {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)